src/console/dlt-passive-node-ctrl.c

```diff
@@ -113,7 +113,7 @@ void set_node_id(char *id)
         exit(-1);
     }
     else {
-        strncpy(g_options.node_id, id, DLT_ID_SIZE);
+        strcpy(g_options.node_id, id);
```

Comment:
This may overwrite the end of array g_options.node_id by one byte. node_id is not handled as a null terminated string. 
strncpy is required to restrict the operation to 4 bytes. DLT IDs are not Null-terminated in dlt-daemon.

Reply:
Thank you, I have restored it. Since there is no length check of argv this is indeed needed.

Reply:
@michael-methner 
Good point for no null-terminated for App and Context ID